src/assets/modules/jquery.status/build/index.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://rawgit.com/qld-gov-au/qg-web-template/Feature/QOL-1101-Maps-JS-Transfer/src/assets/_project/lib/ext/butterfly/butterfly.css">
+  <link rel="stylesheet" href="styles/jquery.status.css">
+</head>
+<body>
+<div class="container">
+  
+</div>
+</div>
+<script
+  src="https://code.jquery.com/jquery-1.12.4.js"
+  integrity="sha256-Qw82+bXyGq6MydymqBxNPYTaUXXq7c8v3CwiYwLLNXU="
+  crossorigin="anonymous"></script>
+
+  <!--<script type="text/javascript" src="https://rawgit.com/qld-gov-au/glue-template/working/src/assets/components/autocomplete/src/lib/generate-id.js"></script>
+  <script type="text/javascript" src="https://rawgit.com/qld-gov-au/glue-template/working/src/assets/_project/lib/ext/butterfly/jquery.butterfly.js"></script>-->
+  <script>
+    //env variables file
+    window.qg = window.qg || {};
+    window.qg.components = window.qg.components || {};
+  </script>
+  <script type="text/javascript" src="https://rawgit.com/qld-gov-au/qg-web-template/working/src/assets/_project/lib/ext/generate-id.js"></script>
```

Comment:
1. We will not be using git urls in the end product. Until this version is deployed onto CDN, we are bound to use git urls.
2. Reduce request to the server: I don't think any other solution would be feasible considering that stand alone modules & web template are interrelated. Please let me know if you know any
3. Managing Modules: It is agreed that modules will be hosted onto CDN for usage. Alternatively modules can be installed through npm. Hence If we make any changes in the module, it is expected to be pushed to CDN or published in npm to view changes in other modules using it.